cypress/features/regression/carouselClassic.feature

```diff
@@ -1,18 +1,18 @@
-Feature: Carousel component
-  I want to change Carousel properties
+Feature: Carousel classic component
+  I want to change Carousel properties for classic story
 
   Background: Open Carousel component page classic
     Given I open "Carousel" component page classic
 
   @positive
   Scenario: Initial slide title is Slide One
-  # When step is run on background as Given
-    Then slide title is "Slide One"
+    # When I open "Carousel" component page classic
```

Comment:
add a comment why it is commented

Reply:
Added "# commented because of BDD default scenario Given - When - Then"